### ISSUE TITLE
Version 1.5.6 doesn't work with RainbowKit

### ISSUE THREAD

**Issue by vladanthales** [rainbow-me/browser-extension#1609]
The latest version (1.5.6) of the Chrome browser extension doesn't work with RainbowKit. It throws the error `window.ethereum.on is not a function`.

![Screenshot_17](https://github.com/rainbow-me/browser-extension/assets/78506565/5c449ee8-9635-429e-9942-f937027b752b)

**Comment by johnnyshankman** [comment]
same issue here

**Comment by johnnyshankman** [comment]
new version auto-updated my browser-extension and is breaking a lot of our front ends at manifold.xyz

we had to put out a weird hot patch this morning to try and fix it but this shouldn't even be an issue imo. why is `on` not there anymore? this is a breaking change not a feature/patch like you put out.

**Comment by DanielSinclair** [comment]
Thank you for spotting. Apologies about the issues We've released a rollback as v1.5.7 that will resolve this problem. A breaking change was not intended, and you will not need to amend your dApp. The change we made in the previous version was intended to resolve a conflict between Coinbase Wallet and Rainbow that appeared in recent weeks. We will follow-up with a resolution for this as well, and will test against the Manifold dApp.
